### ISSUE TITLE
Drakewatcher Manuscripts

### ISSUE THREAD

**Issue by Roguinaa** [kevinclement/SimpleArmory#510]
Any possibility Drakewatcher Manuscripts could be tracked?

Currently there is no way to accurately track which ones you have and not. There's no addon for it, best way is to do it manually. The Manuscript achievements don't have all available manuscripts on them either.

**Comment by nacebomb** [comment]
Yea would be amazing! Need it too

**Comment by Bukama** [comment]
> Currently there is no way to accurately track which ones you have and not. There's no addon for it, best way is to do it manually.

All the things does track them and shows you where you get your missing ones.

**Comment by Roguinaa** [comment]
All The Things is way too much for me. And I barely know how it works, too. I usually end up uninstalling it after a few weeks because of it being so overwhelming.

I'd prefer a list on SA.

**Comment by tcp13** [comment]
> The Manuscript achievements don't have all available manuscripts on them either.

Which manuscripts are missing? I currently track mine in Simple Armory (under Achievements -> Collections -> Dragon Isle Drake Cosmetics) paired with Wowhead comments and Handy Notes addon.

**Comment by Roguinaa** [comment]
All the Forbidden Reach racing ones and a few more.

**Comment by maxdekrieger** [comment]
Unfortunately this information is not exposed by Blizzard via an API such that we can show it on SimpleArmory as far as I know. :(

The information is available in-game of course, and I've seen people using this addon https://www.curseforge.com/wow/addons/manuscripts-journal

**Comment by seirl** [comment]
No, it is available. There is an API for completed quests. Drakewatcher manuscripts are implemented as completed quests, for instance https://www.wowhead.com/spell=380955/deciphering completes the quest 69559. If we wanted to implement that feature, it would be possible to look at all the items flagged as Drakewatcher Manuscripts, get their spells, and get the quest effect, all that from the DB2 files.

Personnally I don't particularly want to implement this feature because it's not account-wide and there are other ways to track them (achievements), but it should be doable. Also possible for Soulshapes.

**Comment by maxdekrieger** [comment]
Oh nice! I didn't know it was possible. Do the achievements really capture them all?

I'm not opposed to implementing such features collectibles, but we need a suitable (not too prominent) place in the ui for this. Would another collectibles menu item be too much? Something like "Other" or "Miscellaneous" for the soulshapes, manuscripts and possible future things as well.

Because we ask for a specific character (name and realm) I don't think it would be crazy to implement character-specific features.

**Comment by Leomelati** [comment]
No, the new Winding Slitherdrake doesn't have any achievements as far as I know.

I couldn't find any Blizzard endpoint that returns if a specific quest was completed, the only thing that I found was a list of all quests completed. For some reason, when I search for quest 69559 or 72367 (Raszageth) in the response I couldn't find any of those even having them.

Using `/run print(C_QuestLog.IsQuestFlaggedCompleted(72367))` ingame returns true. Maybe we will need to get this info by an addon like the old toy page.

**Comment by seirl** [comment]
I would really like not to, the toy addon was a huge hack that I didn't expect to last so many years and I'm glad we finally got rid of it. :stuck_out_tongue:

**Comment by GTmAster** [comment]
I've researched the question a little bit today, and found out that quest API doesn't return tracker quests, among which those learning manuscripts quests. There some requests from community council on this matter (e.g. https://us.forums.blizzard.com/en/wow/t/collection-quest-api/1675734), but now it's not possible. Maybe we will see some API for this type of collectable in 11.0, since they are definitely planning to continue use the system.

**Comment by Rauko** [comment]
Is there any way to track also all new customisations that need to be unlocked like firemoonkin form or new demons for warlock? feels like this is all done as acc-wide quests as soon as all alts of such classes gets in permanently unlocked
